ale_linters/ruby/solargraph.vim

```diff
@@ -0,0 +1,19 @@
+" Author: Horacio Sanson - https://github.com/hsanson
+" Description: Solargraph Language Server https://solargraph.org/
+
+call ale#Set('ruby_solargraph_host', '127.0.0.1')
+call ale#Set('ruby_solargraph_port', '7658')
+
+function! ale_linters#ruby#solargraph#GetAddress(buffer) abort
+    let l:host = ale#VarFunc('ruby_solargraph_host')(a:buffer)
+    let l:port = ale#VarFunc('ruby_solargraph_port')(a:buffer)
```

Comment:
Use `ale#Var(a:buffer, variable_name)` instead. `VarFunc` is just for writing simple callbacks easily.